packages/engine/utils/collectibles.ts

```diff
@@ -0,0 +1,91 @@
+import { modules } from '@walless/ioc';
+import type { CollectibleDocument, CollectionDocument } from '@walless/store';
+
+const getCollectibleById = async (
+	id: string,
+): Promise<CollectibleDocument | undefined> => {
+	return await modules.storage.safeGet(id);
+};
+
+const getCollectionById = async (
+	id: string,
+): Promise<CollectionDocument | undefined> => {
+	return await modules.storage.safeGet(id);
+};
+
+const setCollectible = async (id: string, item: CollectibleDocument) => {
+	return await modules.storage.upsert(id, async () => item);
+};
+
+const setCollection = async (id: string, item: CollectionDocument) => {
+	return await modules.storage.upsert(id, async () => item);
+};
```

Comment:
I prefer name it create or insert, use setTokens makes me think it's a set state action